docs/07Examples/19-EEGProjects/20-Motor_Imagery.md

```diff
@@ -6,7 +6,7 @@ title: Motor Imagery
 This tutorial was made by Rakesh C Jakati.
 
 
-Motor imagery (MI)–based brain-computer interface (BCI) is one of the standard concepts of BCI, in that the user can generate induced activity from the motor cortex by imagining motor movements without any limb movement or external stimulus.
+Motor imagery (MI)–based brain-computing interface (BCI) is one of the standard concepts of BCI, in that the user can generate induced activity from the motor cortex by imagining motor movements without any limb movement or external stimulus.
```

Comment:
A "brain-computer interface" is an accepted term.

https://en.wikipedia.org/wiki/Brain–computer_interface

Easy first pull request. Great job!